app/infrastructure/services.py

```diff
@@ -15,6 +15,7 @@ def search(self, filters: dict) -> Tuple[List[Organization], int, int]:
         page = filters.pop('page')
         page_size = filters.pop('page_size')
         search_text = filters.pop('q')
+        sort = filters.pop('sort', None)
```

Comment:
Should we check for available sorts here? It currently returns a 500 if the sort isn't supported.